x-pack/plugins/security_solution/public/onboarding/hooks/use_stored_state.ts

```diff
@@ -48,6 +52,19 @@ export const useStoredExpandedCardId = (spaceId: string) =>
     null
   );
 
+/**
+ * Stores the selected selectable card ID per space
+ */
+export const useStoredSelectedCardItemId = (
+  selectedCardKey: string,
+  spaceId: string,
+  defaultSelectedCardItemId: CardSelectorListItem['id']
+) =>
+  useDefinedLocalStorage<CardSelectorListItem['id']>(
+    `${selectedCardKey}.${spaceId}`,
+    defaultSelectedCardItemId
+  );
+
```

Comment:
Could we do something like this:

```
export const useStoredSelectedCardItemId = (
  cardType: 'alerts' | 'dashboards' | 'rules',
  spaceId: string,
  defaultSelectedCardItemId: CardSelectorListItem['id']
) =>
  useDefinedLocalStorage<CardSelectorListItem['id']>(
    `${LocalStorageKey.selectedCardItemId}.${cardType}.${spaceId}`,
    defaultSelectedCardItemId
  );
```

So we only have to add one `LocalStorageKey`, and the call is simplified to `useStoredSelectedCardItemId('alerts', spaceId, defaultValue);`

Reply:
great approach, already implemented :)